modules/saml/www/sp/saml2-acs.php

```diff
@@ -252,5 +252,7 @@
 }
 $state['PersistentAuthData'][] = 'saml:sp:prevAuth';
 
+$state['AuthnInstant'] = $assertion->getAuthnInstant();
```

Comment:
Since this is something completely specific to SAML authentication, I think it would make sense to name it `saml:AuthnInstant`, following the same convention we use elsewhere. I also think it should be set before (around line 234), so that it is co-located with other related data.

Finally, you should also consider whether it would be useful for you to have this data later, after authentication happens (and all authproc filters are run). If you think it would, then you would need to consolidate it into the authentication data saved in the session, by adding it to `PersistentAuthData`:

```php
$state['PersistentAuthData'][] = 'saml:AuthnInstant';
```

Reply:
Hi @jaimeperez, 
Thanks for the notes. I'll get the requested changes made. 

As for my needs, accessing it via an authproc filter is fine, but in the hopes of flexibility for others, I think adding it to `PersistentAuthData` is probably a good idea.

Thanks.